yarn.lock

```diff
@@ -11584,7 +11584,12 @@ typescript-eslint@^7.3.1, typescript-eslint@^7.7.1:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.9.5.tgz#095979f9bcc0d09da324d58d03ce8f8374cbe65a"
   integrity sha512-1FXk9E2Hm+QzZQ7z+McJiHL4NW1F2EzMu9Nq9i3zAaGqibafqYwCVU6WyWAuyQRRzOlxou8xZSyXLEN8oKj24g==
 
-typescript@^5.5.0-beta, typescript@~5.5.0-dev.20240327:
+typescript@^5.5.1-rc:
+  version "5.5.1-rc"
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-5.5.1-rc.tgz#4be420f1ecb5e66c52a7cb11444f06f02294c977"
+  integrity sha512-7cbqBhh2QH0RaI7AD0ElJ2Ww/iRdW1w2wH/S2dv6EbdNQQlv39fx+V5VOepxLgfUvRkU5D5pxzgPuvHSAQOdpQ==
+
+typescript@~5.5.0-dev.20240327:
```

Comment:
This dependency on the `dev.20240327` tag comes from `@endo/eslint-plugin`.

Would that cause problems?  If so, maybe a `packageJson.resolutions` entry is in order?  Could you consider creating an issue (or PR) on https://github.com/endojs/endo to fix those dependencies?  Waiting until we update to the official 5.5 release is also acceptable.

Reply:
I don't think it would create problems. 

It'll line up after next sync https://github.com/endojs/endo/pull/2326